climpred/tests/conftest.py

```diff
@@ -237,6 +237,31 @@ def observations_da_1d(observations_ds_1d):
     return observations_ds_1d['SST']
 
 
+@pytest.fixture
+def hindcast_recon_3d(hind_ds_initialized_3d, reconstruction_ds_3d):
+    """HindcastEnsemble initialized with `initialized`, `uninitialzed` and `obs`."""
+    # fix to align coords
+    for c in ['TLAT', 'TLONG', 'TAREA']:
+        reconstruction_ds_3d[c] = hind_ds_initialized_3d[c]
+    hindcast = HindcastEnsemble(hind_ds_initialized_3d)
+    hindcast = hindcast.add_observations(reconstruction_ds_3d, 'recon')
+    hindcast = hindcast - hindcast.sel(time=slice('1964', '2014')).mean('time').sel(
+        init=slice('1964', '2014')
+    ).mean('init')
+    return hindcast
+
+
+@pytest.fixture
+def hindcast_recon_1d_ym(hind_ds_initialized_1d, reconstruction_ds_1d):
+    """HindcastEnsemble initialized with `initialized`, `uninitialzed` and `recon`."""
```

Comment:
`"""HindcastEnsemble with initialized and reconstruction (observations) as an annual time series (no grid)."""`